Frontiers_317_delete_products.py

```diff
@@ -0,0 +1,64 @@
+"""
+Working on a script to clean up demo store. delete all products without images.
+"""
+
+import logging
+import os
+from woocommerce import API
+
+logging.basicConfig(level=logging.INFO)
+
+woo_key = 'WOO_KEY'
+woo_secret = 'WOO_SECRET'
+
+def check_env_variables(variable_name):
+    try:
+        os.environ[variable_name]
+    except KeyError as e:
+        error_message = f"The environment variable must be set: {e}"
```

Comment:
may be you should say what variable is missing. This is generic. You are probably thinking the 'e' will have `KeyError: 'WOO_KEY'`. That is fine but since you are adding a custom message you might as well tell them what variable is missing.